pyproject.toml

```diff
@@ -0,0 +1,33 @@
+[tool.poetry]
+name = "openapi3-parser"
+version = "1.1.16"
```

Comment:
there might be slight issue with this version, previously it was set in `__init__.py` file that is used during publishing process to increment the package version on pypi. If poetry uses this version for a new package for upload, then we should remove redundant package metadata from init file

Reply:
How is the version being bumped now? Is that part of `gh-action-pypi-publish` or manual?

Should for sure only have one source of truth. If automated, could replace with: https://py-pkgs.org/07-releasing-versioning.html#automatic-version-bumping

Reply:
Ah, poetry requires a version so I'll circle back when I can convert setup.py